lib/cfndsl/rake_task.rb

```diff
@@ -0,0 +1,31 @@
+require 'rake'
```

Comment:
@stevenjack looks good...I am happy to update the PR with these changes. Or do you want me to close this PR and create a new one?